### ISSUE TITLE
I am having the same issue, twitch username : Hobos_United

### ISSUE THREAD

**Issue by HobosUnited** [overlay-expert/help-desk#317]
Same issue. I've done everything recommended. Uninstalled, Done through Crome, Logged in/out, etc. 

Please help.

**Comment by HobosUnited** [comment]
> 

Twitch : Hobos_United

**Comment by HobosUnited** [comment]
I have literally done everything mentioned in this article. The issue is not resolved, your attention in this matter is lazy. Please refund me my money immediately- Sent via the Samsung Galaxy S22+ 5G, an AT&T 5G smartphone
-------- Original message --------From: "stale[bot]" ***@***.***> Date: 3/17/22  6:11 PM  (GMT-06:00) To: overlay-expert/help-desk ***@***.***> Cc: HobosUnited ***@***.***>, Author ***@***.***> Subject: Re: [overlay-expert/help-desk] I am having the same issue, twitch username : Hobos_United (Issue #317) 
This issue has been automatically marked as stale because:

the issue template was not used and not enough information was provided
the issue is already answered here or in another issue

Please provide more information, otherwise this issue will be closed if no further activity occurs.

**Comment by marcandrews** [comment]
All you said was "same issue". To assist you, we need to know what issue you are having.

**Comment by HobosUnited** [comment]
Apologies, 
I went to the Creator Dashboard in Twitch. Searched for Overlay Expert, and Installed it. Purchased a $19.99 layered package. Configured it accordingly, and saved it. Went back into My Extensions, and activated the Overlay Expert Extension. The issue arises when I go to select "Choose" on my saved Overlay. It pops up with an error message, and directs me to your help desk. 
I've uninstalled Overlay Expert, and reinstalled it. I've tried every browser possible. Chrome, Mozilla, Internet Explorer, etc. 
I attempted to uninstall again, but now when I search for Extensions the Overlay Expert option is not even available., 
    On Friday, March 18, 2022, 10:52:23 AM CDT, Marc Andrews ***@***.***> wrote:  
 
 


All you said was "same issue". To assist you, we need to know what issue you are having.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
